web/themes/custom/instagram/instagram.theme

```diff
@@ -27,6 +27,9 @@ function instagram_preprocess_page__front(&$variables) {
 
 }
 
+function instagram_preprocess_page__user(&$variables) {
+  $variables['#attached']['library'][] = 'instagram/userpage';
+
```

Comment:
Missing closing brackets

Reply:
this file with change is already done in coding-standards-PR